near-sdk/src/test_utils/context.rs

```diff
@@ -125,8 +125,8 @@ impl VMContextBuilder {
         self
     }
 
-    pub fn block_index(&mut self, block_index: BlockHeight) -> &mut Self {
-        self.context.block_index = block_index;
+    pub fn block_height(&mut self, block_height: BlockHeight) -> &mut Self {
+        self.context.block_height = block_height;
```

Comment:
Let's keep the legacy `block_index` method to avoid a breaking change, mark it deprecated, and add `block_height` as a new one.